zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/GroupController.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.controller.usermanagement;
+
+import io.camunda.identity.usermanagement.CamundaGroup;
+import io.camunda.identity.usermanagement.CamundaUser;
+import io.camunda.identity.usermanagement.service.GroupService;
+import io.camunda.identity.usermanagement.service.MembershipService;
+import io.camunda.zeebe.gateway.protocol.rest.SearchRequestDto;
+import io.camunda.zeebe.gateway.rest.controller.ZeebeRestController;
+import io.camunda.zeebe.gateway.rest.controller.usermanagement.dto.AssignUserToGroupRequest;
+import io.camunda.zeebe.gateway.rest.controller.usermanagement.dto.SearchResponseDto;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ZeebeRestController
+@RequestMapping("/v2")
+public class GroupController {
+  private final GroupService groupService;
+  private final MembershipService membershipService;
+
+  public GroupController(
+      final GroupService groupService, final MembershipService membershipService) {
+    this.groupService = groupService;
+    this.membershipService = membershipService;
+  }
+
+  @PostMapping(
+      path = "/groups",
+      produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_PROBLEM_JSON_VALUE},
+      consumes = MediaType.APPLICATION_JSON_VALUE)
+  @ResponseStatus(HttpStatus.CREATED)
+  public CamundaGroup createGroup(@RequestBody final CamundaGroup group) {
+    return groupService.createGroup(group);
+  }
```

Comment:
❓ Is there a simple get all endpoint defined in the unified api spec? As I remember it (which might be wrong) there are only search endpoints, which do the same if no search filters are passed

Reply:
Theoretically there is `zeebe/gateway-protocol/src/main/proto/rest-api.yaml` which shall contain all endpoint definitions. I will add our things to it now.

Another way of getting it is to use intellij, open a controller, click on the little globe-icon in `@RequestMapping(<globe_icon_here>"/v2/users")` and do `Generate OpenAPI Draft` ... but the `rest-api.yaml` will be updated soon

Reply:
What I mean is: is it intended to have `GET /collection` calls in the unified rest api or only the `POST /collection/search` for the same approach?

Reply:
It is not intended to have `GET /collection` calls in the unified api. The plan is to only have `POST /collection/search` endpoints.